share/completions/brew.fish

```diff
@@ -1,15 +1,56 @@
+function __fish_brew_get_cmd
+  for c in (commandline -opc)
+    if [ (echo $c | string sub -l 1) != "-" ]
```

Comment:
I used echo because when I do like `string sub -l 1 "--test"` I get `string sub: Unknown option '--test'` error.

anyway `string match` is a way better option.